### ISSUE TITLE
Misleading package name.

### ISSUE THREAD

**Issue by folbo** [GetoXs/MediatR.Extensions.FluentValidation.AspNetCore#19]
I don't see the use of "AspNetCore" name included in the Nuget package id as well as the GitHub project name. This project does not depend on AspNetCore at all and in my opinion never will. I feel insecure installing this package in my project which uses MediatR but has nothing to do with AspNetCore. Every time I need to do this my hands are shaking and my eyes start bleeding. Please help.

This package should be named MediatR.Extensions.FluentValidation.

**Comment by GetoXs** [comment]
You are right, unfortunately. Once created, I am unable to change NuGet package. 

Support package renames: https://github.com/NuGet/NuGetGallery/issues/7896
